frontend/app/.server/environment/session.ts

```diff
@@ -28,3 +30,20 @@ export const session = z.object({
   ).pipe(z.number().min(0)),
   SESSION_KEY_PREFIX: z.string().default(defaults.SESSION_KEY_PREFIX),
 });
+
+// Create the session storage object
+const { getSession, commitSession, destroySession } = createCookieSessionStorage({
+  cookie: {
+    name: 'person_case_session', // The name of the cookie
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+    secure: defaults.SESSION_COOKIE_SECURE === 'true', // Convert string to boolean
+    secrets: [defaults.SESSION_COOKIE_SECRET], // Array of secrets used to sign the cookie
+    sameSite: defaults.SESSION_COOKIE_SAMESITE,
+    path: defaults.SESSION_COOKIE_PATH,
+    httpOnly: true, // Prevents JavaScript from accessing the cookie
+    maxAge: Number(defaults.SESSION_EXPIRES_SECONDS), // Convert string to number
+  },
+});
+
+// Export the session functions
+export { getSession, commitSession, destroySession };
```

Comment:
This isn't needed. We're using a server-side session store (redis, when deployed to the cluster).